src/.env.sample

```
@@ -35,3 +35,8 @@ CAZ_PASSWORD=password
 
 # Webapp
 VITE_API_BASE_URL=https://localhost:8000
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)